src/yetibot/core/adapters/slack.clj

```diff
@@ -102,7 +102,7 @@
         (slack-chat/post-message
           (slack-config config) *target* msg
           (merge
-            {:unfurl_media "false" :as_user "true"}
+            {:unfurl_media (str (not (boolean img?))) :as_user "true"}
             (when img?
               {:blocks [{"type" "image"
```

Comment:
Was this intended for another change set?

Reply:
I may have snuck it in :) 
Branch/PR fatigue.